src/components/export-report-button.tsx

```diff
@@ -3,9 +3,10 @@ import cn from 'classnames'
 import { observer } from 'mobx-react-lite'
 
 import { t } from '@i18n'
+import { theme } from '@theme'
 import operationsStore from '@store/operations'
 import mainTableStore from '@store/ws/main-table.store'
-import { Button } from '@ui/button'
+import { Button } from '@ui/button/button'
```

Comment:
Please, create `index.ts` with `export * from './button` in `button` folder and don't change paths.

Reply:
Done